release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -805,6 +805,14 @@ Red Hat does not guarantee backward compatibility for metrics, recording rules,
 
 * You can add and configure additional external Alertmanagers for both platform and user-defined project monitoring stacks.
 * You can disable the local Alertmanager instance.
+* You can assign the new `monitoring-alertmanager-edit` user role to allow non-administrator users to create and silence alerts for default platform monitoring.
```

Comment:
```suggestion
* You should assign the new `monitoring-alertmanager-edit` role to allow non-administrator users to access the Alertmanager UI.

Reply:
I reworded this to emphasize that assigning the new role is required if you want to give non-admin users the ability to create and silence alerts.